src/oss/confluent.sh

```diff
@@ -1497,9 +1497,17 @@ produce_command() {
       export_kafka
       bootstrapserver="--broker-list localhost:${kafka_port}"
     fi
-    removestr="--value-format avro"
-    if [[ "$args" =~ "$removestr" ]]; then
-      args=${args//$removestr/}
+    cloud="--cloud"
+    if [[ "$args" =~ "$cloud" ]]; then
+      file="$HOME/.ccloud/config"
```

Comment:
this is hardcoded here and it won't work if you have two ccloud clusters I believe. ccloud accepts the config as a parameter. Thoughts?

Reply:
I like the simplicity of just specifying the `--cloud` argument.

I'm not against specifying the ccloud config file, but do you happen to know how many users have multiple ccloud clusters?

Reply:
I added this in.  Not super robust, but it works.